src/fontra/views/editor/panel-reference-font.js

```diff
@@ -649,6 +661,44 @@ export default class ReferenceFontPanel extends Panel {
       []
     );
 
+    const addRemoveButtons = createDomElement("add-remove-buttons");
+    addRemoveButtons.addButtonCallback = async () => {
+      // FIXME Add dialog title translation
```

Comment:
When ever we don't have a translation, we add `// TODO: translation` to the code. This way, we can find it easily one day.